marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/steps/CustomStepControllerTest.java

```diff
@@ -75,28 +171,22 @@ void userCanUpdateStep() throws Exception {
             assertEquals(0, orderArtifacts.size(), "should have 0 Order artifacts");
             
             JsonNode customerArtifacts = array.get(0).get("artifacts");
-            assertEquals(1, customerArtifacts.size(), "should have 1 Customer artifact");
-            JsonNode stepData = customerArtifacts.get(0);
-            
-            /* custom step name and entity type should not have been changed */
-            assertTrue(stepData.get("name").asText().equals("myCustomStep"));
-            assertTrue(stepData.get("targetEntityType").asText().equals("http://example.org/Customer-0.0.1/Customer"));
-            
-            /* other fields should have been updated */
-            assertTrue(stepData.get("description").asText().equals("optional"));
-            assertTrue(stepData.get("selectedSource").asText().equals("collection"));
-            assertTrue(stepData.get("sourceQuery").asText().equals("cts.collectionQuery('test')"));
+            assertEquals(2, customerArtifacts.size(), "should have 2 Customer artifacts");

```

Comment:
There is no need to create 2 custom steps here. The main purpose of this test is to make sure when "reader" user gets custom step, the response has "additionalSettings".

Reply:
The purpose of this test is for the endpoint used by Curate tile.  It should fetch all custom steps and their step names.  The other read test is the one that actually tests whether a particular step contains additionalSettings